docs/fixtures/readme.md

```diff
@@ -0,0 +1,75 @@
+# Gosoline Fixture Loader
+While using *gosoline* you are able to define fixtures directly in code. 
+ 
+## Usage
+* Enable fixture loading in your `config.dist.yml` File
+```
+fixtures:
+    enabled: true
+```
+* Make sure to use a custom build tag named `fixtures`
+
+* Define your fixtures in code while using one of the built in `FixtureWriterFactory` and defining a slice of `[]fixtures.FixtureSet{}`.
+For more details check the `/examples/gosoline-fixture-loading` Directory or read the short example below. 
+
+* The `fixtures.FixtureSet{}` has the following definition:
+```
+type FixtureSet struct {
+	Enabled  bool
+	Writer   FixtureWriterFactory
+	Fixtures []interface{}
+}
+```
+You can easily define multiple fixtures to different destinations in one file and enable or disable them for each FixtureWriter.
+
+* Currently there are 4 different FixtureWriterFactories implemented to load fixtures. 
+    * `DynamoDbFixtureWriterFactory`
+    * `DynamoDbKvStoreFixtureWriterFactory` 
+    * `MysqlOrmFixtureWriterFactory` 
+    * `PlainMySqlFixtureWriterFactory` 
```

Comment:
should we adjust this name to MysqlPlainFixtureWriterFactory?

Reply:
Yeah. true